portfolio/src/main/webapp/script.js

```diff
@@ -40,7 +40,26 @@ function viewGame(){
 }
 
 function getGreeting() {
-  fetch('/data').then(response => response.text()).then((name) => {
+  fetch('/data')
+  .then(response => response.text())
+  .then((name) => {
     document.getElementById('greeting').innerText = name;
   });
+}
+
+function getJSONData(){
+  fetch('/data')  
+  .then(response => response.json()) 
+  .then((message) => {
+
+      let messageConntainer = document.getElementById("messages");
+      console.log(message[0]);
```

Comment:
Is there a useful reason for logging this? If so, please add a comment. If not, please remove.

Reply:
I had it, for debugging purposes, but I forgot to delete it